.storybook/src/zoom.stories.ts

```diff
@@ -0,0 +1,285 @@
+import * as d3 from 'd3';
```

Comment:
Try to import only the used method instead of everything in d3

Reply:
Removed import as it was not in use. We need to set up linting.

Reply:
Rebased and fixed d3 imports for ZoomPanHandler